### DISCUSSION TITLE
The first card does not want to appear on page load

### DISCUSSION THREAD

**Discussion by HarleyBerglund** [quasarframework/quasar#11203] [General - Components / Directives / etc] [answered]
I so frustrated right now. I'm building a carousel with some basic cards inside and I can't get the first card to appear on page load. I need to select the card in the navigation for it to appear. Can someone help me with this?

`<template>
  <q-carousel
    v-model="slide"
    transition-prev="slide-right"
    transition-next="slide-left"
    swipeable
    animated
    control-color="white"
    navigation
    padding
    arrows
    class="full-width full-height carousel-container"
  >
    <q-carousel-slide
      class="column no-wrap"
      v-for="info in information"
      :key="info.id"
      :name="info.id"
    >
      <div class="row fit q-gutter-xs q-col-gutter no-wrap flex flex-center">
        <div class="flex flex-center">
          <q-card
            style="
              box-shadow: 0px 0px 40px 0px rgba(0, 0, 0, 0.15);
              width: 500px;

              height: 700px;
            "
            class="product-container"
          >
            <img
              :src="info.image"
              :alt="info.title"
              class="image full-width"
              style="object-fit: cover"
            />

            <div class="q-px-lg q-py-md" style="height: 380px">
              <q-card-section class="full-width" style="height: 70px">
                <p class="title text-center full-width">
                  {{ info.title }}
                </p>
                <p class="date text-center full-width">
                  {{ info.date }}
                </p>
              </q-card-section>
              <q-card-section class="full-width" style="height: 200px">
                <p class="info-text text-center full-width">
                  {{ info.text }}
                </p>
              </q-card-section>
              <q-card-actions class="absolute-center" style="margin-top: 290px">
                <q-btn
                  text-color="white"
                  :ripple="{ color: 'secondary' }"
                  unelevated
                  outline
                  caps
                  ><span style="margin: 12px 60px 12px 60px; font-size: 18px"
                    >Läs mer
                  </span>
                </q-btn>
              </q-card-actions>
            </div>
          </q-card>
        </div>
      </div>
    </q-carousel-slide>
  </q-carousel>
</template>

<script>
import { ref, defineComponent } from "vue";

const information = [
  {
    id: "1",
    title: "RÖJNING UTE PÅ BANAN",
    date: "2021-05-08",
    text: "Idag har hela föreningen haft en stor röjdag, då alla som har tid kan komma ut och bygga på banan, röja sly och allmänt bara hänga där ute. Vi lyckades klippa gräset på speedballbanan som har vuxit vilt under våra inaktiva år. Dagen avslutades med lite korvgrillning.",
    image: "http://vasteraspaintball.se/images/2018/02/22/Bild.jpg",
  },
  {
    id: "2",
    title: "Dagens drop in spel",
    date: "2021-05-09",
    text: "Idag hade vi ett öppet spel då ALLA är välkomna, oavsett om du är nybörjare, spelar för första gången eller är mer erfaren. Det kom hela 21 personer och vi höll på från tidig förmiddag till sen kväll.",
    image: "http://vasteraspaintball.se/images/2018/02/22/Bild.jpg",
  },
  {
    id: "3",
    title: "RÖJNING UTE PÅ BANAN",
    date: "2021-05-10",
    text: "Idag har hela föreningen haft en stor röjdag, då alla som har tid kan komma ut och bygga på banan, röja sly och allmänt bara hänga där ute.Vi lyckades klippa gräset på speedballbanan som har vuxit vilt under våra inaktiva år.Dagen avslutades med lite korvgrillning.",
    image: "http://vasteraspaintball.se/images/2018/02/22/Bild.jpg",
  },
  {
    id: "4",
    title: "Dagens drop in spel",
    date: "2021-05-11",
    text: "Idag hade vi ett öppet spel då ALLA är välkomna, oavsett om du är nybörjare, spelar för första gången eller är mer erfaren. Det kom hela 21 personer och vi höll på från tidig förmiddag till sen kväll.",
    image: "http://vasteraspaintball.se/images/2018/02/22/Bild.jpg",
  },
  {
    id: "5",
    title: "RÖJNING UTE PÅ BANAN",
    date: "2021-05-12",
    text: "Idag har hela föreningen haft en stor röjdag, då alla som har tid kan komma ut och bygga på banan, röja sly och allmänt bara hänga där ute.Vi lyckades klippa gräset på speedballbanan som har vuxit vilt under våra inaktiva år.Dagen avslutades med lite korvgrillning.",
    image: "http://vasteraspaintball.se/images/2018/02/22/Bild.jpg",
  },
];
export default defineComponent({
  setup() {
    return {
      slide: ref(1),
      information,
    };
  },
});
</script>

<style lang="scss" scoped>
.product-container {
  font-family: "Oswald", sans-serif;
  background: $primary;
  color: white;
}

.image {
  height: 320px;
}
.title {
  text-transform: uppercase;
  font-size: 26px;
  line-height: 1;
}
.date {
  font-size: 18px;
  color: $secondary;
  line-height: 0.5;
}
.info-text {
  font-size: 18px;
  color: white;
  line-height: 1.8;
  display: -webkit-box;
  -webkit-line-clamp: 5;
  -webkit-box-orient: vertical;
  overflow: hidden;
}
</style>
`

**Comment by pdanpdan** [comment, marked as answer]
Can you read what you posted?
Your IDs are text and you init the slide to number 1

**Reply by HarleyBerglund** [reply]
omg, I'm so stupid. Thank you so much
